silkrpc/types/transaction.hpp

```diff
@@ -76,6 +76,10 @@ struct CallBundleInfo {
    std::vector<CallBundleTxInfo> txs_info;
 };
 
+struct TransactionInfo {
+   struct Transaction transaction;
```

Comment:
`struct` is redundant here.

BTW, why a struct containing just one field is needed in the first place? You could consider to get rid of `TransactionInfo` and define a type for the whole transaction contents instead (`TransactionContents`). Then you can customize the formatting of transactions in such context by using the `to_json` overload for `TransactionContents`.

Reply:
I have done in this way to permit to call my to_json() with new types and avoid copy fields of transaction after decode()

Reply:
Type is removed